doc/source/release-notes/Percona-XtraDB-Cluster-8.0.26-16.1.rst

```diff
@@ -0,0 +1,48 @@
+.. _PXC-8.0.26-16:
+
+================================================================================
+*Percona XtraDB Cluster* 8.0.26-16
+================================================================================
+
+:Date: 2022
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/8.0/install/index.html>`_
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+Release Highlights
+================================================================================
+
+The following are some of the notable fixes for MySQL 8.0.26, provided by Oracle, and included in this release:
+
+* Deprecation of the `TLSv1 and TLSv1.1 <https://tools.ietf.org/id/draft-ietf-tls-oldversions-deprecate-02.html>`__ connection protocols.
+* Identifiers with certain terms are updated. See `Functionality Added or Changed section in the 8.0.26 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-26.html#mysqld-8-0-26-feature>`__ for a list of updated identifiers. The following terms have been changed:
+    - “master” is changed to “source”
+    - “slave” is changed to “replica”
+    - “mts”, for “multithreaded slave”, is changed to “mta” for “multithreaded applier”.
+
+For more information, see the `MySQL 8.0.26 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-26.html>`__
+
+
+Bugs Fixed
```

Comment:
The following bugs have made it to PXC-8.0.26. I think we should also mention them in the release notes.

1. https://jira.percona.com/browse/PXC-3807
2. https://jira.percona.com/browse/PXC-3779
3. https://jira.percona.com/browse/PXC-3743
4. https://jira.percona.com/browse/PXC-3739
5. https://jira.percona.com/browse/PXC-3776
6. https://jira.percona.com/browse/PXC-3777
7. https://jira.percona.com/browse/PXC-3710

Reply:
Reviewed new list - added what was not internal.